### ISSUE TITLE
Cloning a paste with attachment will result in a paste cannot decrypt

### ISSUE THREAD

**Issue by Haocen** [PrivateBin/PrivateBin#489]
## Steps to reproduce

1. Create a paste with attachment
2. Clone the paste

### What happens
Paste clone should be able to be properly decrypted

### What should happen
Paste cannto be decrypted.

## Basic information

**Server address**:

**Server OS:**
docker-alpine-fpm-nginx

**Webserver:**

**Browser:**
Chrome Version 76.0.3809.100 (Official Build) (64-bit)

**PrivateBin version:**
1.2.1
I can reproduce this issue on <https://privatebin.net>: No, because no file upload enabled.

**Comment by Haocen** [comment]
Error recorded:
```
privatebin.js?1.3:4609 DOMException: Failed to execute 'atob' on 'Window': The string to be decoded is not correctly encoded.
    at Object.jQuery.PrivateBin.me.setAttachment
```

**Comment by Haocen** [comment]
Root cause identified: the `AttachmentViewer.getAttachmentData()` returned `undefined` when trying to encrypt and send cloned paste.

**Comment by Haocen** [comment]
Seemed in order to get orginal blob we need to read the blob with a request:
```
let xhr = new XMLHttpRequest();
xhr.open('GET', 'blob:https://privatebin.net/51b4a5b9-f43d-44d9-84cc-78177994053f);
xhr.responseType = 'blob';
xhr.onload = function(e) {
  if (this.status == 200) {
    var myBlob = this.response;
    // myBlob is now the blob that the object URL pointed to.
  }
};
xhr.send();
```

May need to tweak CSP as well? Not sure about this part.

**Comment by Haocen** [comment]
Fixed and tested on 1.2.1 + Chrome Version 76.0.3809.100 (Official Build) (64-bit)
Required to tweak csp.
Waiting for backport.

**Comment by elrido** [comment]
Oh, wow, so (at least in Chrome) `connect-src *` doesn't include `blob:`? Who would have thought!
